hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -43,6 +44,40 @@
 from hpat.utils import to_array
 
 
+class TypingChecker:
```

Comment:
please add doc string with example of output

Reply:
Done.